frontend/src/static/css/tailwind.css

```
@@ -25,6 +25,432 @@
 @import 'tailwindcss/theme.css' layer(theme);
 @import 'tailwindcss/utilities.css' layer(utilities) important;
 @custom-variant dark (&:where(.dark, .dark *));
+@source "../../../../templates/account";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)